src/utils/axios.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+import { baseURL } from '~/utils/axios.config';
+
+export const axiosInstance = axios.create({
+  baseURL,
+});
```

Comment:
How about adding Axios interceptors here? [Here](https://stackoverflow.com/questions/52737078/how-can-you-use-axios-interceptors) are a couple of examples.

Reply:
Let's move it to cookbook if you don't mind. And put a few examples how to add interceptors for different authorizations.